src/uwtools/drivers/orog_gsl.py

```diff
@@ -80,15 +95,20 @@ def driver_name(cls) -> str:
         return STR.oroggsl
 
     # Private helper methods
+    @property
+    def _input_config_path(self) -> Path:
+        """
+        Path to the input config file.
+        """
+        return self.rundir / "orog_gsl.cfg"
 
     @property
     def _runcmd(self):
         """
         The full command-line component invocation.
         """
-        inputs = [str(self.config["config"][k]) for k in ("tile", "resolution", "halo")]
         executable = self.config[STR.execution][STR.executable]
-        return "echo '%s' | %s" % ("\n".join(inputs), executable)
+        return "%s < %s" % (executable, self._input_config_path.name)
```

Comment:
Here's where I changed the run command.

Reply:
Ah, with the quoted items the `echo` version was probably in trouble. Makes sense.